docker-compose.yaml

```
@@ -28,13 +28,28 @@ services:
     networks:
       - default
 
+  validate:
+    depends_on:
+      postgres:
+        condition: service_healthy
```

```suggestion
  validate-models:
    depends_on:
      migrate:
        condition: service_healthy
```

Otherwise this container exits with `2023-06-29 17:14:57 Error: error reading stores: sql error: ERROR: relation "store" does not exist (SQLSTATE 42P01)`